charts/argocd-apps/templates/events-application.yaml

```
@@ -0,0 +1,29 @@
+{{- if .Values.events.enabled }}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: events
+  namespace: argocd
+spec:
+  destination:
+    namespace: events
+    server: {{ .Values.spec.destination.server }}
+  project: default
+  source:
+    helm:
+      valueFiles:
+      {{- if.Values.dev.enabled }}
+      - dev-values.yaml
+      {{- else }}
+      - values.yaml
+      {{- end }}
```

We should always have the `values.yaml` with `dev-values.yaml` overriding, not sure which order is correct though
```suggestion
      {{- if.Values.dev.enabled }}
      - dev-values.yaml
      {{- end }}
      - values.yaml
```